src/structures/SkyBlock/SkyblockMember.js

```diff
@@ -329,39 +308,33 @@ function getBestiaryLevel(userProfile) {
 
 // eslint-disable-next-line require-jsdoc
 function getSlayer(data) {
-  if (!objectPath.has(data, 'slayer_bosses')) {
-    return null;
-  }
   return {
-    zombie: getSlayerLevel(data.slayer_bosses.zombie),
-    spider: getSlayerLevel(data.slayer_bosses.spider),
-    wolf: getSlayerLevel(data.slayer_bosses.wolf),
-    enderman: getSlayerLevel(data.slayer_bosses.enderman),
-    blaze: getSlayerLevel(data.slayer_bosses.blaze),
-    vampire: getSlayerLevel(data.slayer_bosses.vampire)
+    zombie: getSlayerLevel(data?.slayer?.slayer_bosses?.zombie),
+    spider: getSlayerLevel(data?.slayer?.slayer_bosses?.spider),
+    wolf: getSlayerLevel(data?.slayer?.slayer_bosses?.wolf),
+    enderman: getSlayerLevel(data?.slayer?.slayer_bosses?.enderman),
+    blaze: getSlayerLevel(data?.slayer?.slayer_bosses?.blaze),
+    vampire: getSlayerLevel(data?.slayer?.slayer_bosses?.vampire)
```

Comment:
better do `if (data?.slayer?.slayer_bosses === undefined) { return; }` than spamming `?` here, doesn't rly matter but you should do it like that everything, looks cleaner. we can do that later

Reply:
Optional